_posts/2020-05-15-issue-2.md

```diff
@@ -0,0 +1,71 @@
+
+In Chicago, Instacart shopper and organizer Matthew Telles of [Gig Workers Collective](https://www.gigworkerscollective.org) gives a personal look into the deteriorating working conditions that prompted gig workers to fight back in [an interview for Picture Theory](https://podcasts.apple.com/us/podcast/matthew-telles-gig-workers-are-standing-up-to-big-tech/id1361867735?i=1000473489342).
+
+[Hate groups promoting violence](https://www.metrotimes.com/news-hits/archives/2020/05/11/whitmer-becomes-target-of-dozens-of-threats-on-private-facebook-groups-ahead-of-armed-rally-in-lansing) are continuing to flourish in private Facebook groups. _"Dozens of angry Michiganders, fueled by conspiracy theories and disinformation about the coronavirus, are promoting violence and mobilizing armed rallies against Gov. Gretchen Whitmer on Facebook, in violation of the social media company's policies."_ Facebook's inaction and complacency are unacceptable during this unprecedented public health crisis. Meanwhile, 11,250 current and former content moderators working for Facebook globally &mdash; from [Phoenix](https://www.theverge.com/2019/2/25/18229714/cognizant-facebook-content-moderator-interviews-trauma-working-conditions-arizona) to [the Philippines](https://www.vice.com/en_us/article/ywe7gb/the-companies-cleaning-the-deepest-darkest-parts-of-social-media) &mdash; were awarded [a $52M settlement](https://www.theverge.com/2020/5/12/21255870/facebook-content-moderator-settlement-scola-ptsd-mental-health) of at least $1,000 each. This is a major symbolic victory but a minor cost to Facebook, and a very small amount to compensate for what may be life-long post-traumatic stress for handling some of the worst content on the web.
+
+[Google is rolling back its diversity & inclusion initiatives](https://www.nbcnews.com/news/us-news/current-ex-employees-allege-google-drastically-rolled-back-diversity-inclusion-n1206181) in order to avoid conservative backlash, allege current and former employees. A Google representative claims that it's because Google doesn't have "the global resources" to build that capacity internally, which seems a little hard to believe for a company with a [$100 billion cash hoard](https://www.theverge.com/2019/8/1/20749831/alphabet-google-apple-cash-reserves-richest-company) &mdash; not to mention a pattern of [retaliation against workers](https://www.vox.com/recode/2019/9/9/20853647/google-employee-retaliation-harassment-me-too-exclusive) who are reporting harassment. The tech industry at large has a long history with a severe lack of representation, not only in leadership roles but also in rank-and-file positions. We, as workers, cannot be complacent in this.
```

Comment:
```suggestion
[Google is rolling back its diversity & inclusion initiatives](https://www.nbcnews.com/news/us-news/current-ex-employees-allege-google-drastically-rolled-back-diversity-inclusion-n1206181) in order to avoid conservative backlash, allege current and former employees. A Google representative claims that it's because Google doesn't have "the global resources" to build that capacity internally, which seems a little hard to believe for a company with a [$100 billion cash hoard](https://www.theverge.com/2019/8/1/20749831/alphabet-google-apple-cash-reserves-richest-company) &mdash; not to mention a pattern of [retaliation against workers](https://www.vox.com/recode/2019/9/9/20853647/google-employee-retaliation-harassment-me-too-exclusive) who are reporting harassment. The tech industry at-large has a long history with a severe lack of representation, not only in leadership roles but also in rank-and-file positions. We, as workers, cannot be complacent in this.
```

Reply:
at large => at-large